backup_and_restore/application_backup_and_restore/installing/installing-oadp-intro.adoc

```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="installing-oadp-intro"]
+= Introduction to configuring the OpenShift API for Data Protection
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-cloud
+
+toc::[]
+
+You install the {oadp-first} with your backup storage provider by installing the OADP Operator.
```

Comment:
the preview is not available,
just to clarify:
oadp-first is a doc parameter, correct?

Reply:
**Preview -** 
I am not sure why you cannot see the preview:

* [Introduction to installing OADP ](https://62428--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html)
* [OADP features and plugins](https://62428--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-features-plugins.html)

**_attributes/common-attributes.adoc =>>** 
Yes added
*:oadp-first: OpenShift API for Data Protection (OADP)* - I need to push this through the docs and replace the wording as it should have done when the docs were first done.